scripts/mdx.py

```diff
@@ -0,0 +1,126 @@
+"""
```

Comment:
General question again (sorry 😅) - Do we have a decision point on when we _do_ type python code and not?